osgpkitools/osg-user-cert-renew

```diff
@@ -287,30 +288,22 @@ def process_csr(**arguments):
     connect_retrieve(**arguments)
     return
 
-
-def process_renew(**arguments):
-    """This function would in turn call process csr function"""
-    process_csr(**arguments)
-    return
-
-def extract_info(**arguments):
-    """ Extracts the DN from the certificate provided."""
-    x509 = M2Crypto.X509.load_cert(arguments['usercert'])
+def extract_info(cert):
+    """ Extracts the DN and serial numberfrom the certificate provided."""
+    x509 = M2Crypto.X509.load_cert(cert)
     dec_sid = x509.get_serial_number()
-    hex_sid = "%032X" % dec_sid
-    arguments.update({'serial_number':hex_sid})
-    dn = x509.get_issuer()
-    arguments.update({'DN':dn})
-    return arguments
+    serial_number = "%X" % dec_sid
+    distinguished_name = x509.get_issuer()
+    return distinguished_name, serial_number
 
 if __name__ == '__main__':
     try:
         os.umask(0177)
-        arguments = parse_args()
-        arguments['ssl_context'] = get_ssl_context(arguments['usercert'], arguments['userprivkey'])
-        arguments = extract_info(**arguments)
-        OSGPKIUtils.start_timeout_clock(arguments['timeout'])
-        process_renew(**arguments)
+        ARGS = parse_args()
```

Comment:
Why the name change? It's not a constant.

Reply:
Oops, I should've read the pylint message more closely. Fixed.